### PR TITLE
Maps harvest xml file into temporary arrays for processing in batches

### PR DESCRIPTION
**Pull request by jgreben**

### CODE REVIEW

File: run/do-person-harvest.sh [hunk 1]
```diff
@@ -45,8 +45,19 @@ illiad_date=`/s/sirsi/Unicorn/Bin/transdate -d-1`
 echo "Updating/Inserting keys from /s/SUL/Batchlog/userload.keys.$illiad_date into ILLiad" >> $LOG/harvest.log
 $HOME/run/pop2illiad.sh $illiad_date
 
+# Run harvest.xml.out through folio_api_client ruby script to load users into FOLIO
+cd $FOLIO/lib
+# Split into batches of 5000
+while mapfile -t -n 5000 array && ((${#array[@]}))
+do
+    printf '%s\n' "${array[@]}" > tmp.xml
```
Comment (dlrueda) [inline run/do-person-harvest.sh:53]: should be $OUT/tmp.xml here?

Comment (dlrueda) [reply run/do-person-harvest.sh:53]: and on line 55?

Comment (dlrueda) [reply run/do-person-harvest.sh:53]: guess it’s just a temporary file, but seems better to have it exist somewhere other than the lib directory?  Don’t feel too strongly

Comment (jgreben) [reply run/do-person-harvest.sh]: Good catch. Pushed another commit.

**Review by shelleydoljack — APPROVED** [review]

**Pull request merged**